init.lua

```diff
@@ -425,9 +453,12 @@ local function lootCorpse(corpseID)
                     lootItem(i, getRule(corpseItem), 'leftmouseup')
                 end
             end
+            mq.delay(100)
```

Comment:
similar to above comment may only need like a delay 1 here. is the delay after needed at all?